docs/sources/panels/transformations.md

```diff
@@ -0,0 +1,193 @@
++++
+title = "Transformations"
+type = "docs"
+[menu.docs]
+identifier = "transformations"
+parent = "panels"
+weight = 300
++++
+
+# Transformations
+
+> **Note:** This documentation refers to a Grafana 7.0 beta feature. This documentation will be frequently updated to reflect updates to the feature, and it will probably be broken into smaller sections when the feature moves out of beta.
+
+This page explains what transformations in Grafana are and how to use them.
+
+Transformations process the result set before it’s passed to the visualization. You access transformations in the Transform tab of the Grafana panel editor.
```

Comment:
Some times we refer to panel editor as `panel editor` and sometimes as `Panel editor`. Think we should align on this one.

Reply:
It is the Edit Panel window, so lowercase "panel editor." Will reassess if the name changes in the UI. I thought it was going to change to "Panel edit" in the UI, which was close enough to "Panel editor" for me to justify the capitalization.